### DISCUSSION TITLE
Any update on the next release

### DISCUSSION THREAD

**Discussion by abhishek-ch** [vitobotta/hetzner-k3s#475] [General]
Curious Question, any update on the next release?

**Comment by vitobotta** [comment]
Not sure. I have several things on my list for the next release but the free time is tight at the moment as I am very busy with the day job. Is there a particular feature or something you are looking forward to?

**Reply by abhishek-ch** [reply]
Thanks, i totally understand. I am looking for node label, taint

**Reply by vitobotta** [reply]
That's on the list for the next release :)

**Reply by abhishek-ch** [reply]
Is the new release 2.0.9 support nodeselector label

**Reply by vitobotta** [reply]
Not yet
